### DISCUSSION TITLE
Terraform execution fails for AWS deployment

### DISCUSSION THREAD

**Discussion by Swastik466** [0xPolygon/polygon-edge#950] [Q&A]
Hi,
I am trying to clone and execute terraform to create the stack using the AWS deployment guide :
https://wiki.polygon.technology/docs/edge/get-started/terraform-aws-deployment

But I am getting the following error when creating the lambda resource (while trying to download the function package) 👍 

 Error running command 'curl --ssl -L -o main.zip
│ https://raw.githubusercontent.com/Trapesys/polygon-edge-assm/aws-lambda/artifacts/main.zip': exit status 35. Output:   % Total    %
│ Received % Xferd  Average Speed   Time    Time     Time  Current
│                                  Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
│ --:--:-- --:--:-- --:--:--     0
│ curl: (35) schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check
│ revocation for the certificate.'

Any help is truly appreciated. Thanks in advance.

**Comment by laviniat1996** [comment]
Hi @Swastik466 ,

Thank you for opening this discussion. Unfortunately, this is not really related to Edge. Looks like you might have a problem with the ssl certificate or with the network setup.
